sites/hashdev/src/_pages/blog/2_open-source.mdx

```diff
@@ -0,0 +1,49 @@
+---
+authorPhoto: blog/2_open-source/david.jpg
+postPhoto: blog/2_open-source/OpenSource_Cover.jpg
+title: "Open-source at HASH"
+subtitle: "How our approach to open-source informs what we do"
+author: "David Wilkinson"
+jobTitle: "CEO at HASH"
+date: "May 15th, 2022"
+---
+
+## Working in public
+
+At HASH, we publish as much of our code as possible in public.
+
+Depending on the code in question, it'll typically be released under one of three types of license, each of which has slightly different terms attached to it governing what you can and can't do with the underlying source:
+
+1.  **Open-source and permissive licenses:** in essence these licenses give people the freedom to do pretty much whatever they like, including extend the code and keep it private (making this the most flexible category of license). The entirety of the [Block Protocol](https://github.com/blockprotocol/blockprotocol) is available under these terms (by way of the _MIT License_), alongside most of our blocks and connectors.
+1.  **Open-source and copyleft licenses:** these again give people the freedom to use the code however they like, provided that if they extend or modify the code they contribute those changes back into the open-source as well under the same terms. Our [HASH](https://hash.ai/platform/hash) application is available under the _AGPL_ which guarantees this freedom.
+1.  **Fair-source licenses:** fair-source licenses gives people access to and freedom to use code directly, and the ability to self-host products without reliance on their creators (i.e. us). However fair-source licenses limit the ability of others to create competing commercial offerings based on fair-source code. [HASH Engine](https://hash.ai/platform/engine) is released under the _Elastic License (ELv2)_ which allows anyone to use, copy, distribute, make available, and prepare derivative works. However it restricts folks from using the software to provide a managed service to others (e.g. to provide a cloud simulation SaaS offering).
+
+## How we think about open-source
+
+First, **we contribute** original open-source software (OSS) into the ecosystem through our [HASH platform](https://hash.ai/platform/hash), its connectors, and our blocks. You can [find these](https://github.com/hashintel/hash/tree/main/packages/hash) on GitHub.
+
+Second, **we leverage** existing OSS within our platform to accelerate development of our products and to provide users with best-in-class experiences. Under the hood we rely on great technologies such as OpenSearch (Apache), Snowplow (Apache), Meltano (MIT), dbt (Apache), Great Expectations (Apache), and GitLab (MIT). In addition, we use dozens of other open-source packages within our extended codebase.
```

Comment:
I'd consider swapping the integrations ones out for React, Apollo, NextJS, or some other big ones, in case we end up not using one or more of the ones listed

Reply:
Good call, thanks.